### PR TITLE
adds ZMSCORE to sorted set commands

### PR DESCRIPTION
**Pull request by dominikoehlinger**

### CODE REVIEW

File: lib/redis/distributed.rb [hunk 1]
```diff
@@ -626,6 +626,11 @@ def zscore(key, member)
       node_for(key).zscore(key, member)
     end
 
+    # Get the scores associated with the given members in a sorted set.
+    def zmscore(key, members)
```
Comment (byroot) [inline lib/redis/distributed.rb:630]: Why would the distributed version not take `*members` as well? Sounds like a mistake.

File: test/lint/sorted_sets.rb [hunk 2]
```diff
@@ -293,6 +293,15 @@ def test_zscore
       assert_equal(+Float::INFINITY, r.zscore("bar", "s2"))
     end
 
+    def test_zmscore
+      r.zadd "foo", [1, "s1", 2, "s2"]
+
+      assert_equal [1.0, 2.0], r.zmscore("foo", ["s1", "s2"])
```
Comment (byroot) [inline test/lint/sorted_sets.rb:299]: I don't get it. You defined a variadic method, but here you pass an array.

It should be `r.zmscore("foo", "s1", "s2")`

File: lib/redis/distributed.rb (hunk 1 above)
Comment (dominikoehlinger) [reply lib/redis/distributed.rb]: changed it to variadic parameter

File: test/lint/sorted_sets.rb [hunk 3]
```diff
@@ -293,6 +293,16 @@ def test_zscore
       assert_equal(+Float::INFINITY, r.zscore("bar", "s2"))
     end
 
+    def test_zmscore
+      target_version "6.2.0" do
+        r.zadd "foo", [1, "s1", 2, "s2"]
+
+        assert_equal [1.0, 2.0], r.zmscore("foo", "s1", "s2")
+        assert_equal [1.0, 0.0, 2.0], r.zmscore("foo", "s1", "no_member", "s2")
+        assert_equal [0.0], r.zmscore("no_key", "no_member")#
```
Comment (byroot) [inline test/lint/sorted_sets.rb:302]:
```suggestion
        assert_equal [0.0], r.zmscore("no_key", "no_member")
```

**Comment by byroot** [comment]
This was merged in #1032, apologies for this going stale.

**Pull request closed without merging**